golink.go

```diff
@@ -440,7 +444,16 @@ type expandEnv struct {
 
 	// User is the current user, if any.
 	// For example, "foo@example.com" or "foo@github".
-	User string
+	user string
```

Comment:
Intentional mix if exported/unexported fields?

Reply:
yes.  This was is unexported because we only ever want it accessed through the `User()` func, which triggers the unauthenticated user error.

Reply:
I'll fix the godoc though.